src/supertux/sector_parser.cpp

```diff
@@ -101,6 +101,7 @@ SectorParser::parse_object(const std::string& name_, const ReaderMapping& reader
 void
 SectorParser::parse(const ReaderMapping& sector)
 {
+  GameObjectManager::s_is_loading = true;
```

Comment:
The `s_is_loading` variable seems to be unneeded, since it's set to `true` in `false` in only 2 cases, which both happen within one function. This references all edits to this file.